src/sentry/integrations/bitbucket/webhook.py

```diff
@@ -70,9 +89,8 @@ def __call__(self, organization, event):
         except Repository.DoesNotExist:
             raise Http404()
 
-        if repo.config.get('name') != event['repository']['full_name']:
```

Comment:
This is only one of the pieces of data which could be wrong. Per discussion with @lauryndbrown - rather than check if each possible piece of data is out of date, just push the current data to the DB regardless.

(I still wonder about this - it means we're writing to the DB _every time we see a webhook event_, rather than only in the rare instances when data has changed. It makes for cleaner code, but... maybe this is a case where given that there are only a handful of things to check, better to check them all, even if it's vaguely gross? IDK - will leave it for the moment.)